packages/synchro-charts/src/testing/testing-ground/testingGrounds.tsx

```diff
@@ -10,7 +10,7 @@ export class testingGrounds {
       <div>
         <h3>
           hello, welcome to the{' '}
-          <a href="https://synchrocharts.com/" rel="noreferrer" target="_blank">
+          <a href="https://synchrocharts.com/" rel="noopener noreferrer" target="_blank">
```

Comment:
do we want nofollow here as well?

Reply:
I will look into it. To be honest, i'm not very familiar with these - this was just being surfaced as an error so I did the recommendation :-) 

Will revise accordingly in revision

Reply:
i looked into it breifly and it sounded like nofollow was about seo - since this is an internal testing-ground and not crawlable it seems to me fine as is unless there is some factor I am missing